### ISSUE TITLE
Not able to start subnet network with avalanchego 1.11.0 and subnet-evm 0.6.0

### ISSUE THREAD

**Issue by bhargavkacharla** [ava-labs/subnet-evm#1098]
**Describe the bug**
Existing fuji subnet is not working suddenly and transactions submitted in fuji subnet are pending since a long time, able to send read transactions, but write transactions are pending.
logs from `sudo journalctl -u avalanchego -f`
`INFO peer/peer.go:956 beacon attempting to connect with newer version. You may want to update your client {"nodeID": "NodeID-NpagUxt6KQiwPch9Sd4osv8kD1TZnkjdk", "beaconVersion": "avalanchego/1.11.0"}`
Based on logs, upgraded avalanchego version to 1.11.0 and subnet-evm to 0.6.0 and restarted the network. But getting new error
in the logs,

> 
> Feb 15 17:52:17 ip-172-31-21-192 avalanchego[15897]: ERROR[02-15|17:52:17.192] <2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS Chain> github.com/ava-labs/subnet-evm/core/genesis.go:246: found mismatch between config on database vs. new config storedConfig="{\"byzantiumBlock\":0,\"chainId\":2023032,\"constantinopleBlock\":0,\"contractDeployerAllowListConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\",\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\"],\"blockTimestamp\":0},\"contractNativeMinterConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"eip150Block\":0,\"eip155Block\":0,\"eip158Block\":0,\"feeConfig\":{\"gasLimit\":8000000,\"targetBlockRate\":2,\"minBaseFee\":25000000000,\"targetGas\":15000000,\"baseFeeChangeDenominator\":36,\"minBlockGasCost\":0,\"maxBlockGasCost\":1000000,\"blockGasCostStep\":200000},\"feeManagerConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"homesteadBlock\":0,\"istanbulBlock\":0,\"muirGlacierBlock\":0,\"petersburgBlock\":0,\"subnetEVMTimestamp\":0,\"txAllowListConfig\":{\"adminAddresses\":[\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\",\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"upgrades\":{}}" newConfig="{\"byzantiumBlock\":0,\"chainId\":2023032,\"constantinopleBlock\":0,\"contractDeployerAllowListConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\",\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\"],\"blockTimestamp\":0},\"contractNativeMinterConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"durangoTimestamp\":1707840000,\"eip150Block\":0,\"eip155Block\":0,\"eip158Block\":0,\"feeConfig\":{\"gasLimit\":8000000,\"targetBlockRate\":2,\"minBaseFee\":25000000000,\"targetGas\":15000000,\"baseFeeChangeDenominator\":36,\"minBlockGasCost\":0,\"maxBlockGasCost\":1000000,\"blockGasCostStep\":200000},\"feeManagerConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"homesteadBlock\":0,\"istanbulBlock\":0,\"muirGlacierBlock\":0,\"petersburgBlock\":0,\"subnetEVMTimestamp\":0,\"txAllowListConfig\":{\"adminAddresses\":[\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\",\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"upgrades\":{}}"
> Feb 15 17:52:17 ip-172-31-21-192 avalanchego[15897]: [02-15|17:52:17.193] ERROR chains/manager.go:361 error creating chain {"subnetID": "SEerb4ZQYMPFiSaekS1LqnJqPgYR5uPYBR9ggPjfxc1SaeBSj", "chainID": "2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS", "chainAlias": "2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS", "vmID":"ftkYV1ngCuXQC6rm8GrTkKUD6d2Qsc5kzjWKkAuysyZkiVBsS", "error": "error while creating new snowman vm rpc error: code = Unknown desc = mismatching Durango fork block timestamp in database (have timestamp nil, want timestamp 1707840000, rewindto timestamp 1707839999)"}
> 

**To Reproduce**
1. upgrade existing avalanchego from 1.10.19 to 1.11.0
2. upgrade subnet-evm to 0.6.0
3. sudo systemctl stop avalanchego
4. sudo systemctl start avalanchego
5.  sudo journalctl -u avalanchego -f

**Expected behavior**
subnet to run and start syncing blocks

**Operating System**
Amazon EC2 Ubuntu instance

**Comment by darioush** [comment]
Hi @bhargavkacharla , it seems you updated your node after the network upgrade took place, which is why you are seeing this error.
You can attempt to recover your database by specifying `"skip-upgrade-check": true` as part of the chain configuration and restarting the node.

Note that in the future, is important to upgrade the node's software to the latest version before the network upgrade takes place, since the network upgrade may change the ruleset and if your node accepts blocks that differ from the one in the upgrade, your database will not be recoverable and you would need to sync from an empty database.

**Comment by bhargavkacharla** [comment]
@darioush where can I change the chain configuration to specify  `"skip-upgrade-check": true`

**Comment by darioush** [comment]
The default path for the chain configuration is `~/.avalanchego/configs/chains/<chainID>/config.json`.
The file should contain a valid json object, so if it is empty you should add `{"skip-upgrade-check": true}` to that file.

**Comment by bhargavkacharla** [comment]
In my machine, the path is at `~/.avalanchego/configs/chains/C/config.json`. 
add the required field in json object and now config.json file looks like,
```
{
  "state-sync-enabled": true,
  "skip-upgrade-check": true
}
```
restarted the node
sudo systemctl stop avalanchego
sudo systemctl start avalanchego
sudo journalctl -u avalanchego -f
still getting the same error

**Comment by bhargavkacharla** [comment]
@darioush

**Comment by darioush** [comment]
You would need to put the mentioned configuration in the chain config for your chain. Using information from your logs, the location would be `~/.avalanchego/configs/chains/2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS/config.json`

**Comment by bhargavkacharla** [comment]
@darioush 

there is no such path in my instance,
```
ubuntu@ip-172-31-21-192:~$ cd ~/.avalanchego/configs/chains/2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS/
bash: cd: /home/ubuntu/.avalanchego/configs/chains/2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS/: No such file or directory
```

<img width="1418" alt="Screenshot 2024-02-16 at 12 50 10 AM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/40693c48-73bc-4900-8e4a-b8c2f5c26e7c">

**Comment by bhargavkacharla** [comment]
@darioush 
created the directory and added` {"skip-upgrade-check": true}` to config.json.
getting below error now,

```
Feb 15 19:29:30 ip-172-31-21-192 avalanchego[16644]: [02-15|19:29:30.497] ERROR chains/manager.go:361 error creating chain {"subnetID": "SEerb4ZQYMPFiSaekS1LqnJqPgYR5uPYBR9ggPjfxc1SaeBSj", "chainID": "2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS", "chainAlias": "2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS", "vmID":"ftkYV1ngCuXQC6rm8GrTkKUD6d2Qsc5kzjWKkAuysyZkiVBsS", "error": "error while creating new snowman vm rpc error: code = Unknown desc = failed to validate state while re-processing block (0xda6e59f42f0033738e7a427abed6f7d719f66dddef465a0e7a810a76ac005028: 2111): invalid gas used (remote: 1653562 local: 1654058)"}
Feb 15 19:29:30 ip-172-31-21-192 avalanchego[16644]: [02-15|19:29:30.497] INFO health/worker.go:111 registered new check and initialized its state to failing {"namespace": "health", "name": "2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS", "tags": ["SEerb4ZQYMPFiSaekS1LqnJqPgYR5uPYBR9ggPjfxc1SaeBSj"]}
```

**Comment by darioush** [comment]
Unfortunately this means you will need to delete the node's database and restart syncing from scratch.

**Comment by bhargavkacharla** [comment]
1.  where can I do that ? 
2. since it again syncs from scratch, I can still can get older data after sync right ?
3. how long it will take to sync? block height is 2200

**Comment by bhargavkacharla** [comment]
@darioush

**Comment by darioush** [comment]
- The default location of the avalanchego database is: `~/.avalanchego/db/<network name>` in your case I would recommend moving the database directory, so you have a backup, eg `mv ~/.avalanchego/db/fuji ~/.avalanchego/db/fuji.bak`
- You would be able to get the same data as before.
- It would take a while (several hours) to sync the main network, the 2200 blocks on your chain would likely sync relatively quickly

**Comment by darioush** [comment]
This is normal, first P chain will sync then C, X chains and your subnets chain. 

Each chain is independent so the higher block numbers are from the P chain.

Recommended to wait for the process to complete.

**Comment by bhargavkacharla** [comment]
@darioush 
1. the logs got stopped at this, does it mean it sync is done ?

> 
>     Feb 16 19:11:49 ip-172-31-21-192 avalanchego[382]: [02-16|19:11:49.330] INFO <P Chain> validators/logger.go:50 node added to validator set {"subnetID": "11111111111111111111111111111111LpoYY", "nodeID": "NodeID-GPc1Si7SQ6oAEfP8MzQ3cFN3fsAtkwC6i", "publicKey": "0x", "txID": "ANMB6Nhkapf38GLSM3hUe8rAg4naKMPpujGbdaTvKaoNiFrs8", "weight": 1000000000}
> Feb 16 19:11:49 ip-172-31-21-192 avalanchego[382]: [02-16|19:11:49.340] INFO <P Chain> validators/logger.go:50 node added to validator set {"subnetID": "2J8TLHUDfWuGnEuKKG6n47rkU9YcocEtpxCAaFg447doVBYLEH", "nodeID": "NodeID-GPc1Si7SQ6oAEfP8MzQ3cFN3fsAtkwC6i", "publicKey": "0x", "txID": "xhMY41qkHYUexPpcXgNeuKF4rxrGTb9MF2ksqCGV1zeDE4C88", "weight": 20}
> Feb 16 19:11:49 ip-172-31-21-192 avalanchego[382]: [02-16|19:11:49.340] INFO <P Chain> validators/logger.go:50 node added to validator set {"subnetID": "SEerb4ZQYMPFiSaekS1LqnJqPgYR5uPYBR9ggPjfxc1SaeBSj", "nodeID": "NodeID-GPc1Si7SQ6oAEfP8MzQ3cFN3fsAtkwC6i", "publicKey": "0x", "txID": "q4zcy6vsHWqfKn4ycEVGrn9XVR1itpDYrHvVd6M99uw6bDTwL", "weight": 20}
> Feb 16 19:11:49 ip-172-31-21-192 avalanchego[382]: [02-16|19:11:49.340] INFO <P Chain> validators/logger.go:50 node added to validator set {"subnetID": "25BsTBm8GQZirHUUKoRSjed8NxjTxtnXP1FvGNanV3QRuizkdt", "nodeID": "NodeID-GPc1Si7SQ6oAEfP8MzQ3cFN3fsAtkwC6i", "publicKey": "0x", "txID": "onnhtGXvE2H6mE7EVRbTcLZTnaT84pkSPNiFFcq4trR2rz1VF", "weight": 20}
> Feb 16 19:11:49 ip-172-31-21-192 avalanchego[382]: [02-16|19:11:49.340] INFO <P Chain> validators/logger.go:50 node added to validator set {"subnetID": "dLyxcugrVZarrCSREN4B7idVwoRzEbP3kzyJN2BTsoRqYggN5", "nodeID": "NodeID-GPc1Si7SQ6oAEfP8MzQ3cFN3fsAtkwC6i", "publicKey": "0x", "txID": "2s6Z3cWr4jLW4cZnwGfM7t1DErfamr6DbpgPY81i5Uup4R6jXa", "weight": 20}
> Feb 16 19:11:49 ip-172-31-21-192 avalanchego[382]: [02-16|19:11:49.358] INFO <C Chain> snowman/transitive.go:544 consensus starting {"lastAcceptedBlock": "2wanvW19NMoDDT2jmU7EPkS4UKZ2a9p26e42h1YHpi617nnAmv"}
> Feb 16 19:12:02 ip-172-31-21-192 avalanchego[382]: [02-16|19:12:02.808] INFO <X Chain> snowman/transitive.go:544 consensus starting {"lastAcceptedBlock": "2HfyDoTv6uTTndAE9dvkzRQTjYRV4FuTKwCSsWy7Lf8LYtK3aS"}
> Feb 16 19:12:20 ip-172-31-21-192 avalanchego[382]: INFO [02-16|19:12:20.793] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:1497: Resetting chain preference number=30,036,933 hash=b921f4..8c9d56 drop=1 dropfrom=fd7306..7862ae add=1 addfrom=7aa584..c562ed
> 

Also the status of curl commands is attached,
<img width="818" alt="Screenshot 2024-02-17 at 12 56 59 AM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/976cdd4a-6361-44d6-8d51-be54c1db2602">

2. This is for validator node, am running rpc node as well. does that follow the same steps, backing up db and syncing node from scratch ?

**Comment by bhargavkacharla** [comment]
@darioush

**Comment by darioush** [comment]
- Based on the API output, it seems your node is bootstrapped. 
- You would need to follow the same procedure on the RPC node as well.

**Comment by bhargavkacharla** [comment]
@darioush , validator node and rpc node are synced, but still I cant do any transaction, also the balances for the account is 0 but earlier there were some balance in the accounts

**Comment by bhargavkacharla** [comment]
@darioush I guess the nodes didnt sync for subnets, sync was done only for P, X, C chains

**Comment by bhargavkacharla** [comment]
<img width="1889" alt="Screenshot 2024-02-19 at 8 49 48 PM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/72535a64-c217-49ab-8b9e-bbfddaee4d80">
errors from chainId.log

**Comment by bhargavkacharla** [comment]
getting `nonce too low error` and `execution reverted` while syncing blocks of subnet

**Comment by bhargavkacharla** [comment]
getting older log of config mismatch

> 
> INFO [02-19|06:58:27.675] <2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS Chain> github.com/ava-labs/subnet-evm/trie/triedb/hashdb/database.go:521: Persisted trie from memory database nodes=20 size=2.24KiB time="68.345µs" gcnodes=0 gcsize=0.00B gctime=0s livenodes=0 livesize=0.00B
> ERROR[02-19|06:58:27.680] <2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS Chain> github.com/ava-labs/subnet-evm/core/genesis.go:246: found mismatch between config on database vs. new config storedConfig="{\"byzantiumBlock\":0,\"chainId\":2023032,\"constantinopleBlock\":0,\"contractDeployerAllowListConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\",\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\"],\"blockTimestamp\":0},\"contractNativeMinterConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"eip150Block\":0,\"eip155Block\":0,\"eip158Block\":0,\"feeConfig\":{\"gasLimit\":8000000,\"targetBlockRate\":2,\"minBaseFee\":25000000000,\"targetGas\":15000000,\"baseFeeChangeDenominator\":36,\"minBlockGasCost\":0,\"maxBlockGasCost\":1000000,\"blockGasCostStep\":200000},\"feeManagerConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"homesteadBlock\":0,\"istanbulBlock\":0,\"muirGlacierBlock\":0,\"petersburgBlock\":0,\"subnetEVMTimestamp\":0,\"txAllowListConfig\":{\"adminAddresses\":[\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\",\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"upgrades\":{}}" newConfig="{\"byzantiumBlock\":0,\"chainId\":2023032,\"constantinopleBlock\":0,\"contractDeployerAllowListConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\",\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\"],\"blockTimestamp\":0},\"contractNativeMinterConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"durangoTimestamp\":1707840000,\"eip150Block\":0,\"eip155Block\":0,\"eip158Block\":0,\"feeConfig\":{\"gasLimit\":8000000,\"targetBlockRate\":2,\"minBaseFee\":25000000000,\"targetGas\":15000000,\"baseFeeChangeDenominator\":36,\"minBlockGasCost\":0,\"maxBlockGasCost\":1000000,\"blockGasCostStep\":200000},\"feeManagerConfig\":{\"adminAddresses\":[\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"homesteadBlock\":0,\"istanbulBlock\":0,\"muirGlacierBlock\":0,\"petersburgBlock\":0,\"subnetEVMTimestamp\":0,\"txAllowListConfig\":{\"adminAddresses\":[\"0x88fa61d2faa13aad8fbd5b030372b4a159bbbdfb\",\"0x2505a33a144d9e63c3c53ee65c3108239e04a0af\"],\"blockTimestamp\":0},\"upgrades\":{}}"

**Comment by bhargavkacharla** [comment]
<img width="1489" alt="Screenshot 2024-02-20 at 12 15 57 AM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/38f51f75-59ca-4c90-9689-c43e1e4fec4d">
logs from validator node
@darioush

**Comment by bhargavkacharla** [comment]
@darioush can you please look into this, I am stuck at this as subnet is down

**Comment by darioush** [comment]
It's a bit hard to follow what state and configuration your nodes are in.
- Could you provide more details about how you got the node in the state where it is producing the `found mismatch between config on database vs. new config` message? In any case a workaround for this error is to enable the `skip-upgrade-check` flag like before for that chain.
- Nonce too low is coming from transactions that need to be (re-)issued correctly with the valid nonce that corresponds to the node's state. Since the nonce is lower than the node's, it's likely these transactions were issued before a transaction with the same nonce was accepted by the network. The logs are coming from eth_sendRawTransaction and it's not related to syncing the blocks from the subnet.
- I'm not sure what the last log snippet is showing, is the last log from the validator from 02-16?

**Comment by bhargavkacharla** [comment]
1. removed old db and restarted node, it synced from scratch, while syncing subnet, got this error
3. yes, that is from validator node

**Comment by bhargavkacharla** [comment]
this is running in ec2 instance, am not sure how to send the entire log file from VM

**Comment by ceyonur** [comment]
You can find logs under ~/.avalanchego/logs folder

**Comment by bhargavkacharla** [comment]
[rpc.txt](https://github.com/ava-labs/subnet-evm/files/14342173/rpc.txt)
[validator.txt](https://github.com/ava-labs/subnet-evm/files/14342174/validator.txt)
attached rpc node logs and validator node logs, 
validator node is restarted on feb15 after taking backup of db, it started syncing from scratch
rpc node is restarted on feb19

**Comment by bhargavkacharla** [comment]
chainId: 2BWGibjUxAkYcnWAJo432HBp53FE4j5CRe5bg2hV3j5mfxohnS
subnet: SEerb4ZQYMPFiSaekS1LqnJqPgYR5uPYBR9ggPjfxc1SaeBSj

**Comment by bhargavkacharla** [comment]
@ceyonur

**Comment by bhargavkacharla** [comment]
<img width="1917" alt="Screenshot 2024-02-20 at 3 38 42 PM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/85e44b7d-12e4-4931-8d54-e3678e8fe466">
why is it generating state snapshot since a longtime

**Comment by bhargavkacharla** [comment]
@darioush @ceyonur

**Comment by bhargavkacharla** [comment]
@darioush so do I need to wait for snapshot generation to complete(with eta 47 hours)?

**Comment by bhargavkacharla** [comment]
latest output for` sudo journalctl -u avalanchego -f` 

from validator
<img width="1512" alt="Screenshot 2024-02-21 at 9 12 58 AM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/3c6f8f4f-60f2-4d8f-9948-f34ba23611d7">

from rpc
<img width="1485" alt="Screenshot 2024-02-21 at 9 12 42 AM" src="https://github.com/ava-labs/subnet-evm/assets/154217347/9fee7dc8-13bc-46e1-899a-796d16e68756">

**Comment by darioush** [comment]
The snapshot regeneration does not impact the node (except higher use of CPU and disk), and will continue in the background. 

I'm not sure what account balance you are checking or expecting to be non-zero, but I would recommend checking the same account's balance on a different node.

**Comment by bhargavkacharla** [comment]
some of the account were having some faucet tokens but after removing db and starting node, syncing from scratch all those accounts are showing zero balances

**Comment by bhargavkacharla** [comment]
@darioush, I am running only one rpc node and am just checking the previous deployed contracts and trying to read the state variables in the smart contract, reverting with exceptions, looks like old data of subnet is not available

**Comment by bhargavkacharla** [comment]
@darioush @ceyonur

**Comment by akshayxalts** [comment]
@darioush @ceyonur 

Running into a similar VM version mismatch issue

Please check the following logs and provide an appropriate course of action.

failed to register VM {"vmID": "ftkYV1ngCuXQC6rm8GrTkKUD6d2Qsc5kzjWKkAuysyZkiVBsS", "error": "handshake failed: RPCChainVM protocol version mismatch between AvalancheGo and Virtual Machine plugin. AvalancheGo version v1.11.3 implements RPCChainVM protocol version 35. The VM implements RPCChainVM protocol version 32. Please make sure that there is an exact match of the protocol versions. This can be achieved by updating your VM or running an older/newer version of AvalancheGo. Please be advised that some virtual machines may not yet support the latest RPCChainVM protocol version"}

I have verified that I am using the latest avalanche-cli and avalanchego client versions (1.11.3)

**Comment by ceyonur** [comment]
Hi @akshayxalts 
It seems you're using an older version of subnet-evm with newer avalanchego version. 
Can you open a separate issue in https://github.com/ava-labs/avalanche-cli? This issue is not related to the version mismatch.

**Comment by ceyonur** [comment]
> @darioush @ceyonur
> 
> Running into a similar VM version mismatch issue
> 
> Please check the following logs and provide an appropriate course of action.
> 
> failed to register VM {"vmID": "ftkYV1ngCuXQC6rm8GrTkKUD6d2Qsc5kzjWKkAuysyZkiVBsS", "error": "handshake failed: RPCChainVM protocol version mismatch between AvalancheGo and Virtual Machine plugin. AvalancheGo version v1.11.3 implements RPCChainVM protocol version 35. The VM implements RPCChainVM protocol version 32. Please make sure that there is an exact match of the protocol versions. This can be achieved by updating your VM or running an older/newer version of AvalancheGo. Please be advised that some virtual machines may not yet support the latest RPCChainVM protocol version"}
> 
> I have verified that I am using the latest avalanche-cli and avalanchego client versions (1.11.3)

Hi @akshayxalts 
This issue is not related to the version mismatch. It seems you're using an older version of subnet-evm with newer avalanchego version. You should update your subnete-evm version to v0.6.3.

avalanche-cli might have an issue with providing correct versions. Can you open a separate issue in https://github.com/ava-labs/avalanche-cli?

**Comment by ceyonur** [comment]
We have introduced mandatory network upgrade rescheduling for those chains stuck in the faulty state. See https://docs.avax.network/build/subnet/upgrade/customize-a-subnet#network-upgrades-rescheduling-mandatory-network-upgrades for more information.
